src/Sarif.Driver/Sdk/EntryPointUtilities.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif.Driver.Sdk
+{
+    public static class EntryPointUtilities
+    {
+        internal static string[] GenerateArguments(
+            string[] args,
+            IFileSystem fileSystem,
+            IEnvironmentVariables environmentVariables)
+        {
+            List<string> expandedArguments = new List<string>();
+
+            foreach (string argument in args)
+            {
+                if (argument[0] != '@' &&
+                    (argument.Length == 1 || argument[1] != '@'))
+                {
+                    expandedArguments.Add(argument);
+                    continue;
+                }
+
+                string responseFile = argument.Trim('"').Substring(1);
+                try
+                {
+                    responseFile = environmentVariables.ExpandEnvironmentVariables(responseFile);
+                    responseFile = fileSystem.GetFullPath(responseFile);
+
+                    if (!fileSystem.FileExists(responseFile))
+                        responseFile = null;
+                }
+                catch (IOException)
+                {
+                    responseFile = null;
```

Comment:
>responseFile [](start = 20, length = 12)

Likewise here. Maybe we couldn't open the response file because it was locked or we don't have permission. The tool runs and returns 0, with only the console warning to show for it.